lib/blockscore/candidate.rb

```diff
@@ -19,13 +19,13 @@ def hits
     end
 
     def search(options = {})
-      post "#{api_url}watchlists", options.merge(candidate_id: id)
+      post 'watchlists', options.merge(candidate_id: id)
     end
 
     private
 
-    def resource_member(member)
-      get "#{endpoint}/#{id}/#{member}", {}
+    def resource_member(target)
+      get member_endpoint + target, {}
```

Comment:
I also want to comfirm that appending the `target` to `member_endpoint` is going to result in the expected URL?